gff3tool/lib/ERROR/ERROR.py

```diff
@@ -8,14 +8,9 @@
 
 import sys
 # try to import from project first
-from os.path import dirname
-if dirname(__file__) == '':
-    lib_path = '../lib'
-else:
-    lib_path = dirname(__file__) + '/../lib'
-sys.path.insert(1, lib_path)
+from gff3tool.bin import version
 
-__version__ = '0.0.1'
+__version__ = version.__version__
```

Comment:
It seems to me we only need to include version information in files in `bin` directory.

Reply:
Yep, it can be treated as separate issue. What's your thought on this ? Do you want to deal with this later in internal issue 636 ?

Reply:
Yes, I already link this discussion to issue636.